File: mac/VoiceTests/BudgetTests.swift
import XCTest
@testable import Voice
final class BudgetTests: XCTestCase {
    func testClampAndScale() {
        XCTAssertEqual(Budget.ms(rawChars: 0), 3500)
        XCTAssertEqual(Budget.ms(rawChars: 100), 3700)
        XCTAssertEqual(Budget.ms(rawChars: 2000), 15000)
    }
}

File: windows/Spit.Core.Tests/BudgetTests.cs
namespace Spit.Core.Tests;

/// Port of mac/VoiceTests/BudgetTests.swift.
public sealed class BudgetTests
{
    [Fact]
    public void testClampAndScale()
    {
        Assert.Equal(3500, Budget.Ms(rawChars: 0));
        Assert.Equal(3700, Budget.Ms(rawChars: 100));
        Assert.Equal(15000, Budget.Ms(rawChars: 2000));
    }
}
